app/routes/__marketing/partner.tsx

```diff
@@ -0,0 +1,169 @@
+import { ChevronRightIcon } from "@heroicons/react/20/solid";
+import { ArrowDownCircleIcon, GlobeAltIcon } from "@heroicons/react/24/outline";
+import MarketingButton from "~/components/marketing-button/marketing-button";
+import Footer from "~/features/marketing-shell/footer";
+
+export default function Index() {
+  return (
+    <>
+      <div>
+        <div className="relative flex flex-row">
+          <img
+            src="/img/marketing/partner-bg-testimony.png"
+            className="absolute bottom-0 left-0 -z-10 h-screen w-screen"
+            alt=""
+          />
+          <aside className="hidden lg:block w-1/6 absolute">
+            <div className="fixed top-1/3 left-0 flex flex-col items-center pt-8 pl-5 gap-y-5 -z-10">
+              <div className="w-1 h-20 mx-2.5 mt-7 relative top-0">
+                <div id="pageProgress" style={{ height: `0%` }} className="w-0.5 ml-px h-full bg-black " />
+              </div>
+            </div>
+            <div className="fixed top-1/3 left-0 flex flex-col items-center pt-8 pl-5 gap-y-5 z-10">
+              <a href="#top">
+                <GlobeAltIcon className="text-black h-6 w-6" />
+              </a>
+              <a href="#why">
+                <Circle id="whyCircle" />
+              </a>
+              <a href="#offerings">
+                <Circle id="offeringsCircle" />
+              </a>
+              <a href="#testimony">
+                <Circle id="testimonyCircle" />
+              </a>
+              <a href="#bottom">
+                <ArrowDownCircleIcon className="text-black h-6 w-6" />
+              </a>
+            </div>
+          </aside>
+          <main className="pl-8 pr-12">
```

Comment:
`flex-1` should help center things on larger screen

<img width="1480" alt="image" src="https://user-images.githubusercontent.com/84731260/210989549-7066a8b4-cc8f-476e-b3c5-a9197a18101f.png">

Reply:
Can you give me an example? I can't seem to get it to help